### ISSUE TITLE
About syncnet wt

### ISSUE THREAD

**Issue by wangrui5781** [Rudrabha/Wav2Lip#61]
It worked well when finetune the pretrain generator. But when restarting a new training for generator, the sync loss fix to 6.* or 5.*, and output image is exactly like the random frame, until 60K step and I kill the training. May I continue the training ? or have you tried set syncnet weight to 0.03 from the beginning ?

**Comment by prajwalkr** [comment]
> But when restarting a new training for generator, the sync loss fix to 6.* or 5.*, and output image is exactly like the random frame, until 60K step and I kill the training.

what dataset are you training on?

**Comment by wangrui5781** [comment]
Including LRS2 and CREMA-D, the results are similar.

**Comment by prajwalkr** [comment]
We do not know about CREMA-D. On LRS2, what is your L1 loss? Are you training on the full train set of LRS2?

**Comment by wangrui5781** [comment]
Just a part of LRS2, and the same L1 Loss compared to the repo. I mask whole face with the landmark, and a emotion discriminator to get some expression. I want to improve the result but it can't converge even if I remove the perspectual disc and the emotion disc.

**Comment by wangrui5781** [comment]
And a choice function 
wrong_id = random.randint(self.get_frame_id(img_name) - 30, self.get_frame_id(img_name) - 15) 
in the Dataset class is used to mitigate the gap of train the inference.
When finetuning it always be Okey. So I want to know some training process of the repo such as the step and the L1 loss when sync loss start to converge with the sync_wt is zero or sync loss get down to 0.75.

**Comment by prajwalkr** [comment]
> Just a part of LRS2

How big is this part? How many videos/hours? 

> So I want to know some training process of the repo such as the step and the L1 loss when sync loss start to converge with the sync_wt is zero or sync loss get down to 0.75.

Sync loss starts to reduce gradually in accordance with reduction in L1 loss. When L1 loss goes down to ~0.025, sync loss has come to about 0.75.

We cannot offer you more help though, as your techniques vary quite a bit, such as data sampling, landmark-based masking, additional discriminators and a different underlying training data. There are too many variables where the problem might be.
